README.md

```diff
@@ -37,22 +37,35 @@ Face capture uses the webcam by default for capturing live photos of users. File
 
 In order to start integration, you will need the **API token**. You can use our [sandbox](https://documentation.onfido.com/#testing) environment to test your integration, and you will find the sandbox token inside your [Onfido Dashboard](https://onfido.com/dashboard/api/tokens).
 
-### 2. Generating a JSON Web Token
+### 2. Creating an applicant
 
-For security reasons, instead of using the API token directly in you client-side code, you will need to generate and include a short-lived JSON Web Token (JWT)\* everytime you initialise the SDK. The easiest way to generate a JWT is via the [JWT endpoint](https://documentation.onfido.com/#json-web-tokens) in the Onfido API:
+With your API token, you should create an applicant by making a request to the [create applicant endpoint](https://documentation.onfido.com/#create-applicant) from your server:
 
 ```shell
-$ curl https://api.onfido.com/v2/jwt?referrer=YOUR_WEBPAGE_URL \
+$ curl https://api.onfido.com/v2/applicants \
+    -H 'Authorization: Token token=YOUR_API_TOKEN' \
+    -d 'first_name=John' \
+    -d 'last_name=Smith'
+```
+
+You will receive a response containing the applicant id which will be used to create a JSON Web Token.
+
+### 3. Generating an SDK token
+
+For security reasons, instead of using the API token directly in you client-side code, you will need to generate and include a short-lived JSON Web Token ([JWT](https://jwt.io/)) every time you initialise the SDK. The easiest way to generate an SDK Token is via the [SDK Token endpoint](https://documentation.onfido.com/#sdk-tokens) in the Onfido API:
+
+```shell
+$ curl https://api.onfido.com/v2/sdk_token?referrer=YOUR_WEBPAGE_URL&applicant_id=YOUR_APPLICANT_ID \
```

Comment:
The query string and body are two different parts of an HTTP request. They can both be used to send parameters and Rails parses them both when you access `params` in the controller. There are a few reasons why I think we should use the body here:

1) You have to do URL encoding in the query string: e.g. referrer `https%3A%2F%2F*.onfido.com%2Fdocumentation%2F*`. Sending it in the body is much more readable.
2) We should be consistent with the rest of the API. Whilst this endpoint is not strictly RESTful, our API as a whole is. Developers expect the URL + query string to identify a resource and the body to contain the data to upload to the server. In this case the referrer and applicant ID are uploaded data rather than resources identifiers.
3) It makes it obvious it's a POST request rather than a GET request. GET requests must have an empty body. Sending a POST request with an empty body is a slightly weird use case - notice `curl` does a GET by default with no body and POST by default with a body.
4) Security - the whole request URL will get logged in loads of places. Therefore you need to be careful what you put in it. Typically you have more control over whether the body parameters get logged.